app/src/main/java/com/gustavoas/noti/services/AccessibilityService.kt

```diff
@@ -215,10 +340,27 @@ class AccessibilityService : AccessibilityService() {
     }
 
     private fun isLocked(): Boolean {
-        val keyguardManager = getSystemService(KEYGUARD_SERVICE) as KeyguardManager
         return keyguardManager.isKeyguardLocked
     }
 
+    private val statusBarHeight: Int
+        get() {
+            return ViewCompat.getRootWindowInsets(overlayView)?.getInsets(WindowInsetsCompat.Type.statusBars())?.top ?: 0
```

Comment:
The statusBarHeight is always returning 0 when using the accessibility service

Reply:
i've reverted it to the way it worked before the PR